src/layout/Header/index.tsx

```diff
@@ -70,74 +69,54 @@ export default function Header() {
       <Modal isOpen={showPngBalanceModal} onDismiss={() => setShowPngBalanceModal(false)}>
         {showPngBalanceModal && <PngBalanceContent setShowPngBalanceModal={setShowPngBalanceModal} />}
       </Modal>
-      {isMobile && (
-        <MobileHeader>
-          <MobileLogoWrapper>
-            <Logo height={30} width={140} fillColor={theme.color6} />
-          </MobileLogoWrapper>
-
-          <Box display="flex" alignItems="center">
-            <Web3Status />
-
+      {isMobile ? (
```

Comment:
at 960px width header is looking weird. may be lets keep mobile menu at 960px.
something like this
```
const isMobile = useMedia(`(max-width: ${MEDIA_WIDTHS.upToMedium}px)`)
```